src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -123,5 +129,12 @@ def add_suggestion(kw, suggestion):
             "The DELETE_RUNPATH keyword would clear the runpath directories "
             "between runs. It was removed in 2017 and no longer has any effect.",
         )
+        add_suggestion(
+            "PLOT_SETTINGS",
+            f"The keyword {kw} no longer has any effect, and can be safely removed."
```

Comment:
Maybe we should say "The keyword PLOT_SETTINGS was removed in 2019 and has no effect. It can safely be removed. It was used for controlling settings for outputting qc plots to disk and what plots were shown in the GUI. All plots can now be selected on request from the GUI, and there is also an alternative view accessible by using `ert viz` in the commandline.

Reply:
I like how the feedback actually gives back some information regarding when it was removed, and also what the original purpose was. If we're going nitpick on this, the feedback should be reversed, with explanation first, and wrapping it up with "It was removed in year-no, no longer has any effect and can be safely removed."

And maybe go for either f-string kw substitution or hard-coded string in the next?

Reply:
Maybe, but I also think stating the most relevant information for the user first may help them action on it quicker.